new/detector/implementation/custom/filter.go

```diff
@@ -175,6 +175,12 @@ func matchContentFilter(filter settings.PatternFilter, content string) *bool {
 		return boolPointer(filter.Regex.MatchString(content))
 	}
 
+	if filter.LengthLessThan != nil {
+		if len(content) >= *filter.LengthLessThan {
```

Comment:
I think it would be better to get the string detection to compare against. That way quotes won't be included, and we can support additional cases where variables are used.

In the case there is no string detection, we'd return `nil` - we use ternary logic for the number filters so "not a number" is always falsey, even when you wrap it in `not`.